packages/run-protocol/src/psm/psm.js

```diff
@@ -74,42 +70,38 @@ export const start = async (zcf, privateArgs) => {
   const emptyStable = AmountMath.makeEmpty(stableBrand);
   const emptyAnchor = AmountMath.makeEmpty(anchorBrand);
 
-  // Mock simple goverannce API for parameters that will be controlled by governance
-  const getGovernance = _ => {
-    const {
-      governedParams: { WantStableFeeBP, GiveStableFeeBP, MintLimit },
-    } = zcf.getTerms();
-    assert(
-      AmountMath.isGTE(MintLimit, emptyAnchor),
-      X`MintLimit ${MintLimit} must specify a limit in ${anchorBrand}`,
-    );
-    return {
-      getWantStableRate: () =>
-        makeRatio(WantStableFeeBP, stableBrand, BASIS_POINTS),
-      getGiveStableRate: () =>
-        makeRatio(GiveStableFeeBP, stableBrand, BASIS_POINTS),
-      getMintLimit: () => MintLimit,
-    };
-  };
-  const gov = getGovernance(zcf);
+  const { augmentPublicFacet, makeGovernorFacet, params } =
+    await handleParamGovernance(zcf, initialPoserInvitation, {
+      GiveStableFee: ParamTypes.RATIO,
+      MintLimit: ParamTypes.AMOUNT,
+      WantStableFee: ParamTypes.RATIO,
+    });
 
   const { zcfSeat: anchorPool } = zcf.makeEmptySeatKit();
   const { zcfSeat: feePool } = zcf.makeEmptySeatKit();
   const { zcfSeat: stage } = zcf.makeEmptySeatKit();
 
+  /**
+   * @param {Amount<'nat'>} given
+   */
   const assertUnderLimit = given => {
     const anchorAfterTrade = AmountMath.add(
       anchorPool.getAmountAllocated('Anchor', anchorBrand),
       given,
     );
     assert(
-      AmountMath.isGTE(gov.getMintLimit(), anchorAfterTrade),
+      AmountMath.isGTE(params.getMintLimit(), anchorAfterTrade),
       X`Request would exceed mint limit`,
     );
   };
 
+  /**
+   * @param {ZCFSeat} seat
+   * @param {Amount<'nat'>} given
+   * @param {Amount<'nat'>} wanted
```

Comment:
It looks like `wanted` is optional. Is this the right syntax for that?
```suggestion
   * @param {Amount<'nat'>} [wanted]
```

Reply:
Good catch. Turns out [Typescript infers optional from a default value](https://www.typescriptlang.org/play?filetype=js#code/PQKhFgCgAIWgBADgQwE7ILbQN4GcAuqAlgHYDmAvtANoGoC6UswUAxgPYkHTuL5GdkAG2gBeaAAo6ASjEA+HE2h0AdBxKtk+CdIDcUCvshRe-QUJ1GooCDDhI0mHHVKVlhJiBaR13ACYApgBmyACuQvgBfmKSdKIA5AAWRPGyogrYSqrqmtp6BlaQgSHhkX6WUEA), but this is correct JSDoc.